ledger/ledger.go

```diff
@@ -205,23 +205,20 @@ func openLedgerDB(dbPathPrefix string, dbMem bool) (trackerDBs dbPair, blockDBs
 	var trackerDBFilename string
 	var blockDBFilename string
 
-	commonDBFilename := dbPathPrefix + ".sqlite"
 	if !dbMem {
+		commonDBFilename := dbPathPrefix + ".sqlite"
 		_, err = os.Stat(commonDBFilename)
+		if !os.IsNotExist(err) {
+			// before launch, we used to have both blocks and tracker
+			// state in a single SQLite db file. We don't have that anymore,
+			// and we want to fail when that's the case.
+			err = fmt.Errorf("A single ledger database file '%s' was detected. This is no longer supported by current binary", commonDBFilename)
```

Comment:
can't it cause problems for existing users? How'd they migrate?

Reply:
Yes, it will cause problems. But I'm pretty sure we don't have any such users.
A user that has a single database file need to have created his data directory before March 2019. The first mainnet data directory was created at the beginning of June 2019. I would have developed an upgrade path, but I don't know how to bootstrap such a data directory : we don't have any binary which would produce it.
btw : testnet was created during May 2019, and betanet was created few month after that.